src/Xamarin.Android.Tools.Bytecode/Kotlin/KotlinClassMetadata.cs

```diff
@@ -698,15 +704,15 @@ public enum KotlinPropertyFlags
 		Delegation =		0b10_00_000_0,
 		Synthesized =		0b11_00_000_0,
 
-		IsVar =			0b_000000001_000_00_000_0,
-		HasGetter =		0b_000000010_000_00_000_0,
-		HasSetter =		0b_000000100_000_00_000_0,
-		IsConst =		0b_000001000_000_00_000_0,
-		IsLateInit =		0b_000010000_000_00_000_0,
-		HasConstant =		0b_000100000_000_00_000_0,
-		IsExternalProperty =	0b_001000000_000_00_000_0,
-		IsDelegated =		0b_010000000_000_00_000_0,
-		IsExpectProperty =	0b_100000000_000_00_000_0
+		IsVar =			0b_000000001_00_00_000_0,
+		HasGetter =		0b_000000010_00_00_000_0,
+		HasSetter =		0b_000000100_00_00_000_0,
+		IsConst =		0b_000001000_00_00_000_0,
+		IsLateInit =		0b_000010000_00_00_000_0,
+		HasConstant =		0b_000100000_00_00_000_0,
+		IsExternalProperty =	0b_001000000_00_00_000_0,
```

Comment:
Trying to verify/validate this is redonkulous: 
* https://github.com/JetBrains/kotlin/blob/210cc6704109d70290f2944fb24bcdf58d4f2fae/core/metadata/src/org/jetbrains/kotlin/metadata/deserialization/Flags.java#L22
* https://github.com/JetBrains/kotlin/blob/210cc6704109d70290f2944fb24bcdf58d4f2fae/core/metadata/src/metadata.proto#L443-L444

Fortunately, instead of trying to make sense of their code, I can just have the Java compiler tell me:

```java
import org.jetbrains.kotlin.metadata.deserialization.Flags;

class DumpKotlinFlags {
    public static void main(String[] args) {
        System.out.println("IS_EXTERNAL_PROPERTY=" + Flags.IS_EXTERNAL_PROPERTY.invert(0));
    }
}
```

```sh
% javac -cp ~/Downloads/kotlinc/lib/kotlin-compiler.jar dump-kotlin-flags.java
% java -cp ~/Downloads/kotlinc/lib/kotlin-compiler.jar:. DumpKotlinFlags
IS_EXTERNAL_PROPERTY=16384
```

16384 is 0x4000 which is 0b_0100_0000_0000_0000, which is 0b_001000000_00_00_000_0.  Things match!